src/main/java/com/google/api/codegen/php/PhpDiscoveryContext.java

```diff
@@ -52,13 +54,31 @@
           .put(Field.Kind.TYPE_BOOL, "false")
           .put(Field.Kind.TYPE_INT32, "0")
           .put(Field.Kind.TYPE_UINT32, "0")
-          .put(Field.Kind.TYPE_INT64, "0")
-          .put(Field.Kind.TYPE_UINT64, "0")
           .put(Field.Kind.TYPE_FLOAT, "0.0")
           .put(Field.Kind.TYPE_DOUBLE, "0.0")
+          // As a work-around in Javascript, Disovery importor treats number format strings
+          // as TYPE_INT64 or TYPE_UINT64 depending on the string format.
+          .put(Field.Kind.TYPE_INT64, "\'0\'")
+          .put(Field.Kind.TYPE_UINT64, "\'0\'")
           .build();
 
   /**
+   * A set that contains the method names that have extra suffix in the PHP client code.
+   */
+  private static final ImmutableSet<String> RENAMED_METHODS = ImmutableSet.<String>builder()
```

Comment:
Give an example here. What do we need to rename it to? How is it different from others?